examples/wind_example.ipynb

```diff
@@ -0,0 +1,925 @@
+      "text/plain": [
+       "<Figure size 864x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "monthly_data = ndbc_data['2018']['2018-01']\n",
+    "daily_data = monthly_data['2018-01-10']\n",
+    "\n",
+    "# Remove bad speed data\n",
+    "mean_speed_data = daily_data[daily_data.WDIR < 999]\n",
```

Comment:
This operation is okay. I think this would be better:
`daily_data['WDIR'] = daily_data['WDIR'].replace({999.0:np.NaN})`

I think you should mention the different NaN values when they are first shown up above on the initial import. Maybe replace them at that time.

Also, you change the name to a mean but do not take a mean, is this correct?

Reply:
Good call, I'll add a section about replacing the NaN values with np.NaN and do it up front before creating separate daily and monthly datasets.